templates/generic_show_form.html.twig

```diff
@@ -655,6 +655,16 @@
                         ) }}
                      {% endif %}
 
+                     {% if item.isField('ip_contact') %}
+                        {{ fields.htmlField(
+                           "ip_contact",
+                           item.fields['ip_contact'],
+                           __('Public contact address'),
+                           field_options
+                        ) }}
+                     {% endif %}
```

Comment:
I think this should be placed in `inventory_info.html.twig`. Indeed, it is probably only usefull for dynamic items and moving it there would prevent people that has no `inventory::READ` right to access it.

Reply:
No, user can make ```netinventory``` without doing ```netdiscovery```